rogue-thi-app/data/clubs.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "club": "Studierendenvertretung .",
+    "instagram": "https://www.instagram.com/studverthi/",
+    "website": "https://studverthi.de"
+  },
+  {
+    "club": "Hochschulband .",
+    "instagram": "https://www.instagram.com/studverthi/",
+    "website": "https://studverthi.de"
+  },
+  {
+    "club": "Hochschulkino .",
+    "instagram": "https://www.instagram.com/hochschulkino.thi/",
+    "website": "https://studverthi.de/hochschulkino"
+  },
+  {
+    "club": "think e. V.",
+    "instagram": "https://www.instagram.com/think.thi/",
+    "website": "https://think-thi.de/"
+  },
+  {
+    "club": "Neuland Ingolstadt e. V.",
+    "instagram": "https://www.instagram.com/neuland_ingolstadt/",
+    "website": "https://neuland-ingolstadt.de/"
+  },
+  {
+    "club": "Our Future e. V.",
+    "instagram": "https://www.instagram.com/ourfuture_ingolstadt/",
+    "website": "https://www.ourfuturethi.de/Ne"
+  },
+  {
+    "club": "NEWEXIST e. V.",
+    "instagram": "",
+    "website": "https://newexist.com/"
+  },
+  {
+    "club": "N.I.C.E. e. V.",
+    "instagram": "https://www.instagram.com/newexist_official/",
+    "website": "https://www.thi.de/studium/studentisches-leben/studentische-vereine-an-der-thi/nice-network-international-culture-exchange/"
+  },
+  {
+    "club": "Eta-nol e. V.",
+    "instagram": "https://www.instagram.com/eta_nol_in/",
+    "website": "https://eta-nol.de/"
+  },
+  {
+    "club": "Students' Life e.V.",
+    "instagram": "https://www.instagram.com/studentslife.thingolstadt/",
+    "website": "https://students-life.de"
+  },
+  {
+    "club": "consult.IN e.V.",
+    "instagram": "https://www.instagram.com/consult.in/",
+    "website": "https://consultin.net/"
+  },
+  {
+    "club": "Schanzer Racing Electric e.V.",
+    "instagram": "https://www.instagram.com/schanzerracing/",
+    "website": "https://www.schanzer-racing.de/"
+  },
+  {
+    "club": "UNICEF Hochschulgruppe",
+    "instagram": "https://www.instagram.com/unicef_hsg_in/",
+    "website": "https://www.unicef.de/mitmachen/ehrenamtlich-aktiv/-/hochschulgruppe-thi-ingolstadt"
+  },
+  {
+    "club": "Hochschulgaming Ingolstadt",
+    "instagram": "https://www.instagram.com/hochschulgaming_in/",
+    "website": "https://discord.gg/C67R2yXjJd"
```

Comment:
Hochschulgaming has a website: https://hochschulgaming.de (although it's unfinished)